### PR TITLE
Bug fixes naseer

### PR DESCRIPTION
**Pull request by Naseer-007**
- Bug fixes on input QA page
- Changed button styles to match design
- Added disable button for button QA
- Changed language toggle breakpoints after checking with design

### CODE REVIEW

**Review by xingruoxicic — APPROVED** [review]
Code looks good to me

File: component-library/component-lib/src/lib/language-switch/language-switch.component.ts [hunk 1]
```diff
@@ -29,14 +29,14 @@ export class LanguageSwitchComponent implements OnInit {
     @Inject(PLATFORM_ID) private platformId: object,
     private langToggle: LanguageSwitchButtonService,
     private translate: TranslateService) { 
-      this.isMobile = window.innerWidth <= 360;
+      this.isMobile = window.innerWidth <= 768;
```
Comment (kristopher-houston) [inline component-library/component-lib/src/lib/language-switch/language-switch.component.ts:32]: Please undo the changes here - mobile should be set at 360px

File: component-library/component-lib/src/lib/language-switch/language-switch.component.ts [hunk 2]
```diff
@@ -29,14 +29,14 @@ export class LanguageSwitchComponent implements OnInit {
...
     }
 
-  /** Listens for screen resizes and sets mobile boolean */
+  /** Listens for screen resizes and sets mobile-tablet boolean */
   @HostListener('window:resize', ['$event'])
   handleResize(e: any) {
     if (isPlatformBrowser(this.platformId)) {
-      this.isMobile = window.innerWidth <= 360;
+      this.isMobile = window.innerWidth <= 768;
```
Comment (kristopher-houston) [inline component-library/component-lib/src/lib/language-switch/language-switch.component.ts:39]: Undo

File: component-library/component-lib/src/lib/language-switch/language-switch.component.ts (hunk 1 above)
Comment (kristopher-houston) [reply component-library/component-lib/src/lib/language-switch/language-switch.component.ts:32]: Unless I've missed something

Comment (Naseer-007) [reply component-library/component-lib/src/lib/language-switch/language-switch.component.ts]: breakpoints were changed to tablet breakpoints after checking with sara from design.
